### ISSUE TITLE
Exclusion not being respected for renaming-table rule using CLI

### ISSUE THREAD

**Issue by natalie-zamani** [sbdchd/squawk#166]
A rule exclusion on the `renaming-table` rule was not respected when running the CLI.

Here's the command that I ran (with fake schema files)

```bash
squawk -e=prefer-robust-stmts,adding-field-with-default,adding-not-nullable-field,renaming-table /path/to/V1__schema1.sql /path/to/V2__schema2.sql
```

**Comment by sbdchd** [comment]
do you have an example with some SQL?

I can reproduce with:

```
echo "ALTER TABLE "table_name" RENAME TO "new_table_name";" > foo.sql
squawk -e=renaming-table foo.sql

foo.sql:1:0: warning: renaming-table

   1 | ALTER TABLE table_name RENAME TO new_table_name;

  note: Renaming a table may break existing clients.

find detailed examples and solutions for each rule at https://squawkhq.com/docs/rules
```

on 0.8.0

Also for some reason it's outputting dupe errors with:


```
echo "ALTER TABLE "table_name" RENAME TO "new_table_name";" > foo.sql 
squawk foo.sql
foo.sql:1:0: warning: renaming-table

   1 | ALTER TABLE table_name RENAME TO new_table_name;

  note: Renaming a table may break existing clients.

foo.sql:1:0: warning: renaming-table

   1 | ALTER TABLE table_name RENAME TO new_table_name;

  note: Renaming a table may break existing clients.

find detailed examples and solutions for each rule at https://squawkhq.com/docs/rules
```


I was previously on 0.5.0 and it didn't have this issue so something went wrong

**Comment by sbdchd** [comment]
Should be all fixed in 0.8.1
